pandas/tests/indexes/test_category.py

```diff
@@ -207,19 +207,20 @@ def test_map(self):
         ci = pd.CategoricalIndex(list('ABABC'), categories=list('CBA'),
                                  ordered=True)
         result = ci.map(lambda x: x.lower())
-        exp = pd.Categorical(list('ababc'), categories=list('cba'),
-                             ordered=True)
-        tm.assert_categorical_equal(result, exp)
+        exp = pd.CategoricalIndex(list('ababc'), categories=list('cba'),
```

Comment:
show an example like this in the whatsnew as well (e.g. CategoryIndex.map -> CI rather than Category now)